017-FrontDoor/Coach/Solution03.md

```diff
@@ -0,0 +1,33 @@
+### Notes for Challenge 3 - Offload traffic for a high traffic event
+
+ [< Previous Challenge [2]](./Solution02.md) - **[Home](./README.md)** - [Next Challenge [4] >](./Solution04.md)
+
+The current state of the Front Door that start Challenge 3 will basically have:
```

Comment:
Sentence is a little weird, reword if possible.

Reply:
At the start of Challenge 3, the student's Front Door instance should look like:


(and I'm a little wierd so haha)